src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,259 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exception.DukeException;
+import duke.exception.EmptyDescriptionException;
+import duke.exception.EmptyDateException;
+import duke.exception.WrongCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class Duke {
+
+    static ArrayList <Task> taskArrayList = new ArrayList<Task>();
+    public static String format = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
```

Comment:
String constants should have Uppercase letters in namings: it should be `FORMAT`